common-pipeline/operators/conditions-v3-schema.json

```diff
@@ -599,6 +602,50 @@
       "required": [
         "allow_change"
       ]
+    },
+    "default_value_definition": {
+      "properties": {
+        "default_value": {
+          "description": "Set default value of the parameter_ref if condition evaluates to true.",
+          "type": "object",
+          "properties": {
+            "parameter_ref": {
+              "description": "Parameter whose default value is to be set.",
+              "type": "string",
+            },
+            "value": {
+              "description": "This will be the default value of parameter_ref if condition evaluates to true.",
+              "oneOf": [
```

Comment:
Wouldn't this be any type of value?  For example I think this could also be an object.

Reply:
Take a look at https://stackoverflow.com/questions/16826128/why-is-this-json-schema-invalid-using-any-type.  I think we want all json types supported here.